bin/zuul

```diff
@@ -21,6 +21,7 @@ program
 .option('--tunnel', 'establish a tunnel for outside acceess. only used when --local is specified')
 .option('--phantom', 'run tests in phantomjs. PhantomJS must be installed separately.')
 .option('--tunnel-host <host url>', 'specify a localtunnel server to use for forwarding')
+.option('--sauce-connect [tunnel-identifier]', 'use saucelabs with sauce connect instead of localtunnel. Optionally specify the tunnel-identifier')
```

Comment:
We can't use `zuul --sauce-connect test.js` because test.js is then considered te value of "sauce-connect" option.

Are you using it like so @brycekahle? thanks